core/src/io/anuke/mindustry/ui/dialogs/CustomGameDialog.java

```diff
@@ -94,7 +93,10 @@ void setup(){
             state.wavetime = difficulty.waveTime;
         }).width(s);
 
-        cont.add(sdif).visible(() -> lastPreset != null);
+        cont.add(sdif);//.visible(() -> lastPreset != null);
```

Comment:
Either remove the `visible` code or keep it altogether, don't comment it out if you don't need it.

Reply:
Oh sure, that's an omission.